apps/api/src/applicant/external-api.service.ts

```diff
@@ -343,6 +345,38 @@ export class ExternalAPIService {
     return { users: users, ha: ha };
   }
 
+  async removeMilestonesNotOnATS(
+    applicants: { applicant_id: string; milestones: { id: string }[] }[],
+  ): Promise<void> {
+    let removedCount = 0;
+    await Promise.all(
+      applicants.map(async a => {
+        const audits: { id: string; status_id: string }[] = await getManager().query(`
+        SELECT "audit"."id" id, "status"."id" status_id
```

Comment:
@dbayly-freshworks @coryatwater-FW 
I wanted to use query builder but it doesn't work. The weird thing is that this query is from the debug output of the query builder.